src/Common/CMakeLists.txt

```diff
@@ -11,3 +11,6 @@ endif()
 if (ENABLE_MYSQL)
     add_subdirectory (mysqlxx)
 endif ()
+
+# Otherwise stack traces symbolization is painfully slow in debug build.
+set_source_files_properties("Dwarf.cpp" PROPERTIES COMPILE_FLAGS "-O3")
```

Comment:
I guess `-O2` is enough? (`-O3` make it harder to debug)

Reply:
Yes, most likely it's enough.

Reply:
And there is already this lines - https://github.com/ClickHouse/ClickHouse/blob/d8924f0b0ed96c5b6bc4398f34aa7cc96dcaf8d2/src/CMakeLists.txt#L342-L347